server/server.go

```diff
@@ -80,6 +80,14 @@ func NewServer(config *Config) (*Server, error) {
 		return s.translator.getProxy(s.errorLog, req.URL.Host)
 	}
 
+	// Set both the host and port as system config variables
+	hostname,err := os.Hostname()
```

Comment:
Just check if this will also work in docker.

Reply:
I tested a router to config service call already, using the `whoami` result as the host name, which worked. Will also test the code change.

Reply:
Confirmed: working in docker too